react/react1/week3/reactweek3/src/TodoList.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from "react";
+import TodoItem from "./TodoItem";
+// import { DataFetching } from "./DataFetching";
+
+export function TodoList() {
+  const [useTodo, setUseTodo] = useState([]);
+  const [description, setDescription] = useState("");
+  const [deadline, setDeadline] = useState("");
+  const Api =
+    "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+  useEffect(() => {
+    if (setUseTodo) {
+      fetch(Api)
+        .then((res) => res.json())
+        .then((data) => {
+          const fetchMap = data.map((data) => ({
```

Comment:
There is actually no need to "map" over the returned data.

You only need to use map when you want to modify, add or remove data inside of the array.

Instead you can simply use `setUseTodo(data);`